locale/en/messages.json

```diff
@@ -880,6 +880,7 @@
   "Volumes name": "",
   "Volumes size": "",
   "Waiting": "",
+  "Waiting until the scheduler task is up and running.": "",
```

Comment:
Did we run this copy by anyone in UX?

Reply:
@nLight suggested it.

Reply:
@izharbuendia could you please validate the message? Thanks!